src/main/scala/com/databricks/labs/overwatch/pipeline/Pipeline.scala

```diff
@@ -425,6 +427,33 @@ class Pipeline(
     Helpers.getDatesGlob(startDate.toLocalDate, untilTime.asLocalDateTime.plusDays(1).toLocalDate)
   }
 
+  //Get the dataframe from either tempDirectory or from the ResponseArray
+  private def getApiData(apiTempPath: Option[String]): DataFrame ={
+     //Get the data from either tempDirectory or from the ResponseArray
+     val tmpClusterEventsSuccessPath = s"${config.tempWorkingDir}/${apiTempPath.get}/success" + config.runID
+    val tmpClusterEventsErrorPath = s"${config.tempWorkingDir}/${apiTempPath.get}/error" + config.runID
+    null
+  }
+
+  //Add moduleID to the dataframe
+  private def addModuleID(apiDF: DataFrame):DataFrame = {
+    null
+  }
```

Comment:
this is a simple withColumn, I don't think we need a whole func for this

Reply:
agreed